libbeat/common/mapstr.go

```diff
@@ -138,7 +149,13 @@ func (m MapStr) GetValue(key string) (interface{}, error) {
 // to insert values (e.g. m[key] = value).
 func (m MapStr) Put(key string, value interface{}) (interface{}, error) {
 	// XXX `safemapstr.Put` mimics this implementation, both should be updated to have similar behavior
```

Comment:
Is the comment above still relevent?

Reply:
yes, comment is still relevant. reminds me, I have to update `safemapstr.Put` :/

Reply:
Heh It was an indirect request 👿 
I will merge when Jenkins has finished the dance, ping me if I forget.